dask/sizeof.py

```diff
@@ -54,6 +52,13 @@ def sizeof_python_collection(seq):
         return getsizeof(seq) + sum(map(sizeof, seq))
 
 
+@sizeof.register(set)
+@sizeof.register(frozenset)
+def sizeof_python_sets(seq):
+    # As of Python v3.9, it is deprecated to call random.sample() on sets
+    return sizeof_python_list_and_tuple(list(seq))
```

Comment:
Nitpick: we tend to favor letting `sizeof` handle dispatching instead of calling a specific `sizeof_*` implementation. This will be more robust to future changes like, for example, function name changes. 

```suggestion
    return sizeof(list(seq))
```